src/pages/index.js

```diff
@@ -10,6 +11,7 @@ export default class IndexPage extends Component {
     render() {
         return (
             <div>
+                <Navbar/>
```

Comment:
Também de acordo com o [style guide do airbnb](https://github.com/airbnb/javascript/tree/master/react#spacing), seria "correto" utilizar o seguinte espaçamento:

```suggestion
                <Navbar />
```